modules/ROOT/pages/troubleshoot-agent-network.adoc

```diff
@@ -0,0 +1,44 @@
+= Troubleshoot Your Agent Network
+
+Agent Visualizer gives you quick access to tools to troubleshoot your agent network.
+
+== Viewing Metrics
+
+. Select a node to open the detail card.
+. In *Performance and Health*, click *View metrics* to access the metrics for the node:
++
+* Successful Requests
++
+This metric is shown for brokers, agents, and MCP servers. This is the total number of requests received successfully in the last 24 hours. 
+* Failed Request Rate
++
+This metric is shown for brokers, agents, and MCP servers. This is the percentage of requests that failed in the last 24 hours. 
+* A2A Calls Sent
++
+This metric is shown for brokers. This is the total number of messages sent to agents using the Agent2Agent (A2A) protocol in the last 24 hours. 
+* MCP Calls Sent
++
+This metric is shown for MCP servers. This is the total number of messages sent to  MCP servers in the last 24 hours. 
+
+== Accessing Logs
+
+On US and EU Clouds, Anypoint Monitoring aggregates log files so you can manage, search for, filter, and analyze your logs. You can use aggregated logs in conjunction with monitoring to help identify issues and investigate failures quickly. 
```

Comment:
Should "monitoring" be init cap here?

Reply:
can you put an anchor over the subject so I can link to it?